deploy/demo/quickstart/README.md

```diff
@@ -0,0 +1,87 @@
+# Quickstart Demo - AAD Pod Identity
+
+The quickstart demo for AAD Pod Identity is a single script designed to steamline and expedite the installation and setup of AAD Pod Identity for a demo and/or POC with a minimum amount of file editing and Azure infrastructure needed to showcase it's core functionality. If you are unfamiliar with the concept of AAD Pod Identity, it may be best to visit the main [Getting Started](https://github.com/Azure/aad-pod-identity#getting-started) page, and review all the concepts.
+
+## Getting Started
+> **NOTE: This quickstart makes the following assumptions:**
+> * You currently have an AKS cluster deployed and it is set to your current context
+> * You will be deploying the azureidentity and azureidentitybinding in the K8 default namespace
+> * Your AKS cluster is RBAC enabled
+> * You have admin config of the AKS cluster
+> * You will be using a Managed System Identity and not a Managed User Identity
+> * The latest Azure CLI installed
+> * jq and sed installed | This script was designed to be run in a Linux BASH environment
+
+> If any of the assumptions are not true, it may best best to opt for the full lenght [AAD Pod Tutorial](https://github.com/Azure/aad-pod-identity/tree/master/docs/tutorial#aad-pod-identity-tutorial) or reference the [Getting Started](https://github.com/Azure/aad-pod-identity#getting-started) page.
+
+### 1. Edit the aad-pod-identity-demo-config.json file
+The aad-pod-identity-demo-config.json file is the only file you will have to edit to get AAD Pod Identity quickstart demo deployed. Please open the json file and edit the following properties:
+
+* ASSETSRESOURCEGROUPNAME
```

Comment:
These JSON fields are more readable if they are in `camalCase`. What do you think?

Reply:
I agree